.gitattributes

```diff
@@ -3,3 +3,7 @@
 # https://github.com/github/linguist.
 
 go.sum linguist-generated=true
+
+# To avoid git status and other tools reporting files due to different line endings
+# normalize the line endings to LF
+* text=auto eol=lf
```

Comment:
I am not sure I understand what that this do, can you explain if this means we will have a mix of both and we just ignore diffs?

Reply:
This is something to make developing on Windows a bit smoother: the default git configuration on Windows converts `lf` to `crlf`, but then tools like lint and fmt report/change line endings in many source files. Preserving the lf on checkout even on Windows avoids that, in practice nowadays the tools on Windows support `lf` just fine.

Let me try some better wording on the comment to make the reason clear.

Reply:
[b8544c8](https://github.com/open-telemetry/opentelemetry-collector/pull/13625/commits/b8544c8e4b43f04e8214382fe45c0175d25be86f)